### DISCUSSION TITLE
Welcome to CockroachDB Discussions!

### DISCUSSION THREAD

**Discussion by kenliu** [cockroachdb/cockroach#59182] [General]
## 👋 Welcome!
  We're testing out Discussions as an alternative to our [forum](https://forum.cockroachlabs.com/), for folks to ask questions, share their CockroachDB accomplishments, and connect with other people in the community. We'll be trying out Discussions for a few months to see how it works for us (or doesn't).

Make sure to also join our [Community Slack](https://go.crdb.dev/p/slack) while you're here!

We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas and projects you're working on with CockroachDB.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with CockroachDB.

**Comment by jordanlewis** [comment]
👋 I work at Cockroach Labs, on our SQL system. I'm excited to try out this Discussions forum and hopefully answer lots of questions!

**Reply by SWDevAngel** [reply]
Hi Everyone! I'm Lisa - DevRel at Cockroach Labs! Really psyched about the new GitHub Discussions. We hope to see lots of engagement here! Also, let us know what YOU think of this new feature. :)

**Comment by SaikrishnaBijja** [comment]
Hello, I am new to open source. i want to contribute to CockroachDB, i know python and some basics in golang, i can anyone help me where to start

**Reply by rafiss** [reply]
I'm glad you'd like to contribute! Take a look at this for some ideas on getting started: https://github.com/cockroachdb/cockroach/issues/41815

**Comment by pouyasadeqpour** [comment]
N

**Comment by Yash-xoxo** [comment]
Y
